buildrpm/python-drgn-tools.spec

```diff
@@ -61,6 +61,9 @@ rm %{buildroot}/usr/bin/DRGN
 %{_mandir}/man1/corelens.1.gz
 
 %changelog
+* Tue Aug 27 2024 Stephen Brennan <stephen.s.brennan@oracle.com> - 1.1.0-1
```

Comment:
should this be 1.1.0 instead of 1.1.0-1?

Reply:
No, most RPMs have a release version appended with the hyphen, it starts at 1. If we update the packaging alone, we'll update that number.

Reply:
It comes from line 5 above.